### PR TITLE
[Media] Markdown help text 

### PR DESCRIPTION
**Pull request by llevitis**
This uses the React "Markdown" component to add help text to media/help/ for both the media main page and the edit media files subpage. 

This PR can be used in conjunction with #3078 or instead of it.

### CODE REVIEW

File: modules/media/help/media.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Media
+
+The Media module serves as a repository for files associated with a particular candidate-timepoint or instrument in a study.  Users can access and upload files such as PDF scans, recordings, log files, or stimulus presentation media in formats such as .pdf, .mp4, .mp3, and .txt. 
+
+## Searching for a Media File
```
Comment (driusan) [inline modules/media/help/media.md]: This (and the file above) are inconsistent in if there's a space under the header or not. Sometimes there's an empty line, sometimes there's not.

File: modules/media/help/media.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Media
+
+The Media module serves as a repository for files associated with a particular candidate-timepoint or instrument in a study.  Users can access and upload files such as PDF scans, recordings, log files, or stimulus presentation media in formats such as .pdf, .mp4, .mp3, and .txt. 
+
+## Searching for a Media File
+Under the Browse tab, use the Selection Filters to search for files by fields such as including filename, file type, candidate PSCID, Visit Label, Instrument, Site, or name of user who uploaded the file.  Partial string matching can be used on many fields (e.g. filename).  As filters are selected, the below data table will dynamically update with relevant results.  Click the “Clear Filters” button to reset all filters.
```
Comment (driusan) [inline modules/media/help/media.md]: "including" shouldn't be there in the first sentence. The capitalization of what you can filter by is also inconsistent (I think they should all be lower case, they're not proper nouns.)

File: modules/media/help/edit.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Edit Media File
+This page allows users to modify information about an uploaded file.  Fields such as Date of Administration, Comments, and file visibility (hidden or visible) can be updated.  Note that fields such as PSCID, Visit Label, Site, and Instrument cannot be modified on uploaded media files.
```
Comment (driusan) [inline modules/media/help/edit.md]: Site and Instrument aren't proper nouns, they should be lower case.

File: modules/media/help/media.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+The Media module serves as a repository for files associated with a particular candidate-timepoint or instrument in a study.  Users can access and upload files such as PDF scans, recordings, log files, or stimulus presentation media in formats such as .pdf, .mp4, .mp3, and .txt. 
+
+## Searching for a Media File
+Under the Browse tab, use the Selection Filters to search for files by fields such as including filename, file type, candidate PSCID, Visit Label, Instrument, Site, or name of user who uploaded the file.  Partial string matching can be used on many fields (e.g. filename).  As filters are selected, the below data table will dynamically update with relevant results.  Click the “Clear Filters” button to reset all filters. 
+
+Within the data table, results can be sorted in ascending or descending order by clicking on any column header. To download a file, click on the filename (in blue text). 
+
+## Uploading a Media File
+Under the “Upload” tab, users will be able to upload a new media file. 
+Note that a file to be uploaded should follow the naming convention listed at the top of the Upload page: **[PSCID]****_****[VisitLabel]****_****[Instrument]**
```
Comment (driusan) [inline modules/media/help/media.md]: Why are there so many stars? Isn't one on each side enough?

File: modules/media/help/media.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Users must specify information about the file by selecting from a number of dropdown menus including PSCID, Visit Label, Site, Instrument (optional), Date of Administration, and enter comments on the file in the textbox provided. Lastly, users will select the file they wish to upload by clicking the “Browse” button and selecting the file from their computer. To begin the upload, users must click “Upload File”.
```
Comment (driusan) [inline modules/media/help/media.md]: I don't think "a number of" is required in this sentence. "by selecting from dropdown menus" flows better.. (also same issues with capitalization).

"Lastly, users will select" is also kind of awkward. Maybe just "Next, users select.." or something of the sort would flow better.

File: modules/media/help/media.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+Within the data table, results can be sorted in ascending or descending order by clicking on any column header. To download a file, click on the filename (in blue text). 
+
+## Uploading a Media File
+Under the “Upload” tab, users will be able to upload a new media file. 
+Note that a file to be uploaded should follow the naming convention listed at the top of the Upload page: **[PSCID]****_****[VisitLabel]****_****[Instrument]**
+
+Users must specify information about the file by selecting from a number of dropdown menus including PSCID, Visit Label, Site, Instrument (optional), Date of Administration, and enter comments on the file in the textbox provided. Lastly, users will select the file they wish to upload by clicking the “Browse” button and selecting the file from their computer. To begin the upload, users must click “Upload File”. 
+
+## Editing a Media File
+Click “Edit” from the “Edit Metadata” column to update or edit certain file properties, via the “Edit Media File” page.
```
Comment (driusan) [inline modules/media/help/media.md]: There's a Shatner comma before "via" that shouldn't be there.

File: modules/media/help/edit.md (hunk 3 above)
Comment (christinerogers) [reply modules/media/help/edit.md]: They're capitalized here to reflect the field labels printed in the form.

File: modules/media/help/media.md (hunk 2 above)
Comment (christinerogers) [reply modules/media/help/media.md]: thanks, good catch.  In general, where a Selection Filter field is referenced by its label (usually capitalized), that will be reflected in the help text for easy identification by the user.

File: modules/media/help/edit.md (hunk 3 above)
Comment (driusan) [reply modules/media/help/edit.md]: In that case they should have quotation marks.

File: modules/media/help/edit.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+# Edit Media File
+
+This page allows users to modify information about an uploaded file.  Fields such as Date of Administration, Comments, and file visibility (hidden or visible) can be updated.  Note that fields such as PSCID, Visit Label, "Site", and "Instrument" cannot be modified on uploaded media files.
```
Comment (driusan) [inline modules/media/help/edit.md]: should "PSCID" and "Visit Label" have quotes too? (same with all the other places that columns are referenced in media.md)

Comment (driusan) [reply modules/media/help/edit.md]: also "Date of Administration", "Comments", "file visibility"..

**Pull request closed without merging**